### PR TITLE
Implement the thing

### PR DESCRIPTION
**Pull request by trieloff**
Depends on https://github.com/adobe/helix-shared/pull/328 (hard) and https://github.com/adobe/helix-shared/pull/327 (soft, for new features)

### CODE REVIEW

**Comment by trieloff** [comment]
@tripodsan I think the CI Trigger needs a token

**Comment by tripodsan** [comment]
> @tripodsan I think the CI Trigger needs a token

I added it.

File: src/index.js [hunk 1]
```diff
@@ -13,15 +13,59 @@ const { wrap } = require('@adobe/openwhisk-action-utils');
 const { logger } = require('@adobe/openwhisk-action-logger');
 const { wrap: status } = require('@adobe/helix-status');
 const { epsagon } = require('@adobe/helix-epsagon');
+const RedirectConfig = require('@adobe/helix-shared/src/RedirectConfig');
```
Comment (tripodsan) [inline src/index.js:16]: shouldn't we export it properly?

File: src/index.js [hunk 2]
```diff
@@ -13,15 +13,59 @@ const { wrap } = require('@adobe/openwhisk-action-utils');
...
+  const config = await new RedirectConfig()
+    .withRepo(owner, repo, ref)
+    .init();
+
+  const match = await config.match(path);
+
+  if (match && match.type === 'temporary') {
+    return {
+      statusCode: 302,
+      body: `moved temporarily <a href="${match.url}">here</a>`,
```
Comment (tripodsan) [inline src/index.js:42]: do we need to care about XSS here?

Comment (trieloff) [reply src/index.js:42]: The pedantic answer is yes. I'll just remove the links.

File: src/index.js (hunk 1 above)
Comment (trieloff) [reply src/index.js:16]: depends on https://github.com/adobe/helix-shared/pull/330

**Pull request merged**